SYCL/ESIMD/private_memory/pm_access_1.cpp

```diff
@@ -0,0 +1,12 @@
+//==--------------- pm_access_1.cpp - DPC++ ESIMD on-device test ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// TODO enable on Windows and Level Zero
+// REQUIRES: linux && gpu && opencl
```

Comment:
Please check with @DenisBakhvalov if `REQUIRES: linux` can be removed - can be done as successive patch.

Reply:
Today, unfortunately, we cannot remove this restriction. We can revisit this ~2 weeks from now.